src/FindIntrinsics.cpp

```diff
@@ -384,6 +384,7 @@ class FindIntrinsics : public IRMutator {
             auto is_x_same_int = op->type.is_int() && is_int(x, bits);
             auto is_x_same_uint = op->type.is_uint() && is_uint(x, bits);
             auto is_x_same_int_or_uint = is_x_same_int || is_x_same_uint;
+            auto x_y_same_sign = (is_int(x) && is_int(y)) || (is_uint(x) && is_uint(y));
```

Comment:
I think a comment here would help-- it's not necessarily clear at first glance what `is_int()` means compared to `Type::is_int()`